src/components/Restart.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const Button = ({ render, restart }: RestartProps) => {
```

Comment:
```suggestion
const Button = ({ hidden, restart }: RestartProps) => {
```

I'd definitely be more explicit with this prop name, render is usually used when passing JSX

Reply:
it's probably not necessary to put this in it's own component either (you're only using it in one place and it doesn't have any state), just conditionally render it in the return you're using it